src/pam-ssh-add/pam-ssh-add.c

```diff
@@ -809,9 +809,11 @@ static int
 stash_password_for_session (pam_handle_t *pamh,
                             const char *password)
 {
-  if (pam_set_data (pamh, STORED_AUTHTOK, strdup (password),
+  char *password_copy = strdup (password);
```

Comment:
check for null or use the "x" variant, please

Reply:
Good catch, done.